.github/workflows/ci-win.yml

```diff
@@ -51,20 +51,27 @@ jobs:
         path: hadoop-winutils
         ref: d018bd9c919dee1448b95519351cc591a6338a00
 
+    - name: Pull Hadoop
+      # A local Hadoop setup is required on Windows, very sad...
+      uses: nick-fields/retry@v2
+      with:
+        max_attempts: 10
+        timeout_minutes: 5
+        retry_wait_seconds: 3
+        command: |
+          cd $HOME
+          Invoke-WebRequest -Uri https://archive.apache.org/dist/hadoop/common/hadoop-3.3.2/hadoop-3.3.2.tar.gz -OutFile hadoop-3.3.2.tar.gz
```

Comment:
we could use these here instead:
```
      [-MaximumRetryCount <Int32>]
      [-RetryIntervalSec <Int32>]
```
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7.3

also maybe we should prefer to download from the dlcdn instead of the archive:
https://dlcdn.apache.org/hadoop/common/hadoop-3.3.2/hadoop-3.3.2.tar.gz

Reply:
updated

Reply:
hmm sorry seems like the new url doesnt work ... even though `-MaximumRedirection` is 5 by default :thinking:

Reply:
probably a missing `cd $HOME`

Reply:
that was it